app/src/main/java/org/mozilla/fenix/settings/logins/PasswordRevealHelper.kt

```diff
@@ -0,0 +1,41 @@
+package org.mozilla.fenix.settings.logins
+
+import android.content.Context
+import android.text.InputType
+import android.widget.ImageButton
+import android.widget.TextView
+import org.mozilla.fenix.R
+import org.mozilla.fenix.components.metrics.Event
+import org.mozilla.fenix.ext.components
+
+fun togglePasswordReveal(
+    context: Context?,
```

Comment:
You can just get the context from the views: `passwordText.context`. That way it doesn't need to be nullable.

Reply:
Sorry but I didn't get what you mean

Reply:
```
fun togglePasswordReveal(
    passwordText: TextView,
    revealPasswordButton: ImageButton,
    password: String? = null
) {
  val context = passwordText.context // never null :)
```

Reply:
oh my god, great

Reply:
@NotWoods also now using ContextCompat.getDrawable. This will not cause issue, right? No problem on local build.

Reply:
Github having problems, commit not seen in here.
https://www.githubstatus.com/

Reply:
There's a lot of methods for getting drawables, but the best is AppCompatResources.getDrawable as it uses the vector compat library. For some reason ContextCompat does not.

Reply:
Thanks, updated.

Reply:
I wonder when will github just work.